plugins/modules/openssl_privatekey.py

```diff
@@ -9,79 +9,75 @@
-        type: str
-        default: present
-        choices: [ absent, present ]
-    force:
-        description:
-            - Should the key be regenerated even if it already exists.
-        type: bool
-        default: false
-    path:
-        description:
-            - Name of the file in which the generated TLS/SSL private key will be written. It will have V(0600) mode
-              if O(mode) is not explicitly set.
-        type: path
-        required: true
-    format:
-        version_added: '1.0.0'
-    format_mismatch:
-        version_added: '1.0.0'
-    backup:
-        description:
-            - Create a backup file including a timestamp so you can get
-              the original private key back if you overwrote it with a new one by accident.
-        type: bool
-        default: false
-    return_content:
-        description:
-            - If set to V(true), will return the (current or generated) private key's content as RV(privatekey).
-            - Note that especially if the private key is not encrypted, you have to make sure that the returned
-              value is treated appropriately and not accidentally written to logs etc.! Use with care!
-            - Use Ansible's C(no_log) task option to avoid the output being shown. See also
-              U(https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-keep-secret-data-in-my-playbook).
-        type: bool
-        default: false
-        version_added: '1.0.0'
-    regenerate:
-        version_added: '1.0.0'
+  state:
+    description:
+      - Whether the private key should exist or not, taking action if the state is different from what is stated.
+    type: str
+    default: present
+    choices: [absent, present]
+  force:
+    description:
+      - Should the key be regenerated even if it already exists.
+    type: bool
+    default: false
+  path:
+    description:
+      - Name of the file in which the generated TLS/SSL private key will be written. It will have V(0600) mode if O(mode)
+        is not explicitly set.
+    type: path
+    required: true
+  format:
+    version_added: '1.0.0'
+  format_mismatch:
+    version_added: '1.0.0'
+  backup:
+    description:
+      - Create a backup file including a timestamp so you can get the original private key back if you overwrote it with a
+        new one by accident.
+    type: bool
+    default: false
+  return_content:
+    description:
+      - If set to V(true), will return the (current or generated) private key's content as RV(privatekey).
+      - Note that especially if the private key is not encrypted, you have to make sure that the returned value is treated
+        appropriately and not accidentally written to logs etc.! Use with care!
+      - Use Ansible's C(no_log) task option to avoid the output being shown. See also U(https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-keep-secret-data-in-my-playbook).
```

Comment:
```suggestion
      - Use Ansible's C(no_log) task option to avoid the output being shown. See also
        U(https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-keep-secret-data-in-my-playbook).
```